CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to Bicep
+We are very happy to accept community contributions to Bicep, whether those are [Pull Requests](#pull-requests), [Example Files](#example-files), [Feature Suggestions](#feature-suggestions) or [Bug Reports](#bug-reports)! Please note that by participating in this project, you agree to abide by the [Code of Conduct](./CODE_OF_CONDUCT.md), as well as the terms of the [CLA](#cla).
+
+## Getting Started
+* If you haven't already, you will need [dotnet](https://dotnet.microsoft.com/download) 3.1 (or later) and [node + npm](https://nodejs.org/en/download/) 10 (or later) installed locally to build and run this project.
+* You are free to work on Bicep on any platform using any editor, but you may find it quickest to get started using [VSCode](https://code.visualstudio.com/Download) with the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp).
+* Fork this repo (see [this forking guide](https://guides.github.com/activities/forking/) for more information).
+* Checkout the repo locally with `git clone git@github.com:{your_username}/bicep.git`.
```

Comment:
>bicep [](start = 75, length = 5)

Did you mean to update this one? Not sure if it's case-sensitive.

Reply:
Yeah, I realized the repo was lower-case B, so I changed this. Not sure if that matters, but may as well get it right.